tools/ci_build/github/azure-pipelines/templates/win-web-ci.yml

```diff
@@ -131,9 +131,15 @@ jobs:
       key: onnxtestdata | $(Build.SourcesDirectory)\js\scripts\prepare-onnx-node-tests.ts
       restoreKeys: |
         onnxtestdata | $(Build.SourcesDirectory)\js\scripts\prepare-onnx-node-tests.ts
-        onnxtestdata
       path: $(Build.SourcesDirectory)/js/test/
+      cacheHitVar: CACHE_RESTORED
     displayName: 'Cache ONNX node test data'
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'
+      script: find "$(Build.SourcesDirectory)/js/test/" -type f
```

Comment:
does this actually work? it expends to something like `find "C:\source\1\a/js/test/" -type f`

Reply:
It works because it's a Bash Task
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1063199&view=logs&j=4cf9212c-8936-5e77-cbdb-290c1c5567eb&t=382f73a8-ec47-5d0a-d520-3509e6c4458a

CI agents support Bash, Batch and Powershell on Windows. It'd better to use one language to reduce maintenance cost.